app/modules/individuals/models.py

```diff
@@ -269,6 +321,118 @@ def _consolidate_social_groups(self, source_individual):
                 f"merge passing membership to {socgrp} from {source_individual} [roles {data.get('roles')}]",
             )
 
+    # this might end up being a SiteSetting etc
+    @classmethod
+    def get_merge_request_deadline_days(cls):
+        return 14
+
+    # does the actual work of setting up celery task to execute this merge
+    # NOTE: this does not do any notification of users; see merge_request_from()
+    def _merge_request_init(self, individuals, parameters=None):
+        from app.modules.individuals.tasks import execute_merge_request
+        from datetime import datetime, timedelta
+
+        if not individuals or not isinstance(individuals, list) or len(individuals) < 1:
+            msg = f'merge request passed invalid individuals: {individuals}'
+            AuditLog.backend_fault(log, msg, self)
```

Comment:
Why is this a backend fault? THis is the request we're sending to the backend, It's our fault, or probably the users, in other words houston or frontend fault

Reply:
oh yeah; i was abusing `backend_fault` ... did a bunch of fixing up of that now and am only using them where its certain the edm was involved.